### DISCUSSION TITLE
Strange values after a while (iOBroker+MQTT Adapter, MPPT 1000/15)

### DISCUSSION THREAD

**Discussion by fat-randy** [softwarecrash/Victron2MQTT#13] [Q&A]
Hello,
I'm using Victron2MQTT (pre-assembled and flashed hardware DIY solderless kit) with my my solarcharger Victron 100/15. At the beginning it worked very well. I received the data via the MQTT adapter in iOBroker. But after a while the victron2mqtt "is writing strange entries into my adapter":
![image](https://github.com/softwarecrash/Victron2MQTT/assets/163835071/3a16ccdf-cf2f-403a-91ac-1cf6c65cdca3)
I'm using the delivered 5V charger to supply the Victron2MQTT Adapter via USB (Don't remember the state of the jumper for power source) . Currently the device is offline, I need to check it in detail when I'm back at home.
Anybody faced the same issues?
Please let me know. Thanks in advance.

**Comment by softwarecrash** [comment]
this data garbage comes up when the esp crash or reboot or lose wifi connection while it send data, so the iobroker mqtt server is not the best option.
check your wifi strength

**Comment by fat-randy** [comment]
Hallo Rocky,

ich würde gerne das W-Lan wechseln. Leider sehe ich im Webpanel keine 
Möglichkeit das zu machen.

Wie gehe ich da vor?

Bitte um Rückmeldung.

Vielen Dnak.

Am 2024-04-02 13:49, schrieb Rocky:

**Reply by softwarecrash** [reply]
bring den ESP außer reichweite des bisherigen WLAN oder resete ihn

**Comment by fat-randy** [comment]
Vielen Dank.

Am 2025-01-31 09:31, schrieb Rocky:
